_posts/2022-06-06-homebrew-3.5.0.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 3.5.0
+author: MikeMcQuaid
+redirect_from: /blog/3.5.0/
+---
+Today I'd like to announce Homebrew 3.5.0. The most significant changes since 3.4.0 are improved `brew update` behaviour and Homebrew (on macOS) requiring at least OS X El Capitan (10.10).
+
+Major changes and deprecations since 3.4.0:
+
+- [`brew update` now defaults to `HOMEBREW_UPDATE_REPORT_ONLY_INSTALLED` behaviour, showing only information on installed formulae, so `HOMEBREW_NO_UPDATE_REPORT_ONLY_INSTALLED` was added instead](https://github.com/Homebrew/brew/pull/13299). This improves performance and usability of `brew update`.
+- [`brew update` lists "Outdated" rather than "Updated" formulae by default](https://github.com/Homebrew/brew/pull/13375). It was already calculating which formulae were outdated and this information is more useful than showing which formulae were changed and is significantly faster than doing version comparisons.
```

Comment:
Shouldn't this be the reverse? That is,
```suggestion
- [`brew update` lists updated rather than outdated formulae by default](https://github.com/Homebrew/brew/pull/13375). It was already calculating which formulae were outdated and this information is more useful than showing which formulae were changed and is significantly faster than doing version comparisons.
```

Reply:
@carlocab Nope. I'm trying to clarify that the "Updated" heading isn't shown by default and instead we show "Outdated". Thoughts on how to clarify this? Check out the PR if it helps!

Reply:
Hmm, ok. Not totally sure. It still seems confusing, as this looks like a change that was made after 3.4 but before 3.5, so doesn't really capture what's changed since 3.4. Does that make sense?

Reply:
No, sorry 😅